benchmarks/benchmark_test.go

```diff
@@ -0,0 +1,85 @@
+package benchmarks
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/bradfitz/gomemcache/memcache"
+	"github.com/daangn/minimemcached"
+)
+
+const (
+	key   string = "testKey"
+	value string = "testValue"
+)
+
+func BenchmarkMemcached(b *testing.B) {
+	mc := memcache.New("127.0.0.1:54008")
+
```

Comment:
I am wondering if there will be a significant change when putting `b.ResetTimer()` here.

Reply:
On my machine, adding `b.ResetTimer()` didn't seem to make any significant changes at all.